### ISSUE TITLE
How to specify paths with whitespaces in Unix?

### ISSUE THREAD

**Issue by EliuX** [meganz/MEGAcmd#151]
In my Windows CI I am executing this command and works right away
```
mega-put -c "target/folder with white spaces/yolo.txt" "/megacmd4j/remote folder/"
```
Nonetheless, when I run this in a Ubuntu xenial distro in Travis CI, [I get an error](https://github.com/EliuX/MEGAcmd4J/issues/13): in my Mega.nz drive I get created a folder like
> home > travis > build >EliuX > MEGAcmd4J > " > megacmd4j > remote folder

`/home/travis/build/EliuX/MEGAcmd4J` is the folder where my Travis CI code runs. Somehow, it seems to have appended that working folder to whatever it sees after the `" ` at the end of the local file.
I also got the same result with
```
mega-put -c target/folder\ with\ white\ spaces/yolo.txt "/megacmd4j/remote folder/"
```

I saw that in the read me this example was given for the `mega-put` command
> mega-put /path/to/my/temporary_resource /exportedstuff/

but I have not seen yet any example like those I have tested, i.e., using paths with whitespaces. 

Long story short, I would like to know how can I work with local and remote paths with whitespaces, working with MEGAcmd in Unix based Operative systems, like Ubuntu.

## Environment used for tests
OS name: "linux", version: "4.15.0-1023-gcp", arch: "amd64", family: "unix"
Linux Distro: Ubuntu
Build group: stable
Build dist: xenial
penjdk version "11.0.1" 2018-10-16
OpenJDK Runtime Environment 18.9 (build 11.0.1+13)
OpenJDK 64-Bit Server VM 18.9 (build 11.0.1+13, mixed mode)
MEGA CMD version: 1.0.0: code 1000000
MEGA dependencies:
* libc-ares2 
* libcrypto++9v5 
* libmediainfo0v5 
* libzen0v5

Thanks in advance.

**Comment by polmr** [comment]
Hi @EliuX , apologies for the late response ...
Both examples (quoted and escaped) should have worked. There must be something we are missing ... Where are you executing your commands? In a terminal? which shell, bash?
Also, let's see a more verbose output to see what's being passed to MEGAcmd Server. Please, try and execute this:
```
mega-put -c -vvv "target/folder with white spaces/yolo.txt" "/megacmd4j/remote folder/"
```

**Comment by polmr** [comment]
ummm in that case, there might be something missing here. You can increase the log level to verbose:
```
mega-log -c VERBOSE
```
and see the log output to view exactly what is MEGAcmd Server receiving. My guess is some _unescaping_ is being performed in the other application's codebase. You can check if that command run directly from bash runs ok, and then try to debug the differences.

**Comment by EliuX** [comment]
Thanks, that worked for me to debug with more detail
